synapse/lib/layer.py

```diff
@@ -2654,10 +2654,11 @@ async def syncIndexEvents(self, offs, matchdef, wait=True):
             for buid, form, edit in editses:
                 for etyp, vals, meta in edit:
                     if ((form in formm and etyp in (EDIT_NODE_ADD, EDIT_NODE_DEL))
-                            or (etyp in (EDIT_PROP_SET, EDIT_PROP_DEL) and (vals[0] in propm
-                                                                            or f'{form}:{vals[0]}' in propm))
+                            or (etyp in (EDIT_PROP_SET, EDIT_PROP_DEL)
+                                and (vals[0] in propm or f'{form}:{vals[0]}' in propm))
```

Comment:
would it potentially worth memoizing this string construction to cut down on new objects in the tight loop?